### PR TITLE
Introduce --newest for repoclosure

### PR DESCRIPTION
**Pull request by j-mracek**
It allows to run repoclosure on all packages and not only on latest.

### CODE REVIEW

**Comment by ignatenkobrain** [comment]
@j-mracek feel free to merge it

File: doc/repoclosure.rst [hunk 1]
```diff
@@ -40,9 +40,15 @@ Options
 ``--arch <arch>``
     Query only packages for specified architecture, can be specified multiple times (default is all architectures).
 
+``--best``
+    Check only the newest packages per arch
```
Comment (ignatenkobrain) [inline doc/repoclosure.rst]: fullstop please.

File: doc/repoclosure.rst [hunk 2]
```diff
@@ -40,9 +40,15 @@ Options
...
+
 ``--check <repoid>``
     Specify repo ids to check, can be specified multiple times (default is all enabled).
 
+``--newest``
+    Check only the newest packages in the repos
```
Comment (ignatenkobrain) [inline doc/repoclosure.rst]: fullstop please.

**Comment by mluscon** [comment]
merged ;-)

**Comment by mluscon** [comment]
Sometimes things that looks good break tests.

**Pull request closed without merging**